RELEASE-PROCESS.rst

```diff
@@ -1,13 +1,14 @@
 For maintainers of Luigi, who have push access to pypi. Here's how you upload
 Luigi to pypi.
 
-#. Make sure [twine](https://pypi.org/project/twine/) is installed ``pip install twine``.
-#. Update version number in `luigi/__meta__.py`.
+#. Make sure [uv](https://github.com/astral-sh/uv) is installed ``curl -LsSf https://astral.sh/uv/install.sh | sh``.
```

Comment:
Here we suggest to install uv with curl+sh, but in `contributing.rst`, we suggest `pipx`. We should be consistent between the two docs. Personally, i prefer curl+sh because it doesn't require the installation of another tool to install uv.

Reply:
fixed in [6f8a76f](https://github.com/spotify/luigi/pull/3338/commits/6f8a76f86d48df9fee67469dfe079bd97a911338)